### DISCUSSION TITLE
AG Server not loading on Mac 10.13.6

### DISCUSSION THREAD

**Discussion by Baldix80** [apohl79/audiogridder#635] [Help]
Hi guys, I'm new here.
Already downloaded and installed AG Server and Plugin both 1.1.1 for 10.8+.

I'm on macOS HS 10.13.6 with a Mac Pro 5.1

As described in the title, when I launch AG Server it doesn't startup, anything happens and anything shows up.

Yesterday, at first launch, it started and scanned my plugins, but after that, program never started. And I can't load it.

Anyone can help me, please?

Thank you in advance!

Baldix80

**Comment by apohl79** [comment]
AG server is a background app and puts an icon into the system tray. I suggest you watch the demo video on the website to get an idea about how to use it.
